### ISSUE TITLE
[Addic7ed] No Subtitle downloaded for a TV Show

### ISSUE THREAD

**Issue by MoonArrow** [Diaoul/subliminal#918]
Hi, I use subliminal for years now on my home made automatic process to get subtitles. Thanks for that first because it's a very valuable piece of software 👍 
Since few days, my backlog of TVshows is growing because of missing subtitles. At first I tought that they are simply not ready but I isolated the problem to one TV Shows not recognised by Addic7ed (the service I use). I simply can't confirm that other shows are impacted or not but this one I explore as much as possible. So, the show exists on Addic7Ed and subtitles are available (http://www.addic7ed.com/show/6388)
so, please below the request and the output I got for one particular episode (obviously user/password have been changed but checked)

```
# subliminal  --debug  --addic7ed "myuser" "mypassword"  download --force -l en --min-score 35 --encoding utf-8 -p addic7ed Alex.Inc.S01E04.HDTV.x264-SVA.mkv
Collecting videos  [####################################]  100%  Alex.Inc.S01E04.HDTV.x264-SVA.mkvDEBUG:subliminal.cli:Collecting path Alex.Inc.S01E04.HDTV.x264-SVA.mkv
INFO:subliminal.core:Scanning video 'Alex.Inc.S01E04.HDTV.x264-SVA.mkv' in ''
DEBUG:subliminal.core:Size is 174987115
DEBUG:subliminal.core:Computed hashes {'thesubdb': '140e2b8a069502bfff49e3baf5dff296', 'opensubtitles': '85aa8823d9724b67', 'shooter': '9d71d3ca38b80fdc6936e8b64ca72159;9dcb3f9d006781ea7a7ebd61c8cd8b02;79ec156df3124e6278969b5865e5c3ac;cf95bf325f3e5e25957c37faacf83aee', 'napiprojekt': '2ae4e0db0cd62218ce9f5eb8bb2745c7'}
INFO:subliminal.core:Refining video with metadata
DEBUG:subliminal.refiners.metadata:Found video_codec h264
DEBUG:subliminal.refiners.metadata:Found audio_codec AAC
DEBUG:subliminal.refiners.metadata:MKV has no subtitle track
INFO:subliminal.core:Refining video with tvdb
INFO:subliminal.refiners.tvdb:Searching series 'Alex Inc'
DEBUG:subliminal.refiners.tvdb:Found 1 results
ERROR:subliminal.refiners.tvdb:No matching series found
INFO:subliminal.core:Refining video with omdb
WARNING:subliminal.refiners.omdb:No results for series
Collecting videos  [####################################]  100%
1 video collected / 0 video ignored / 0 error
Downloading subtitles  [####################################]  100%  Alex.Inc.S01E04.HDTV.x264-SVA.mkvINFO:subliminal.core:Listing subtitles with provider 'addic7ed' and languages set([<Language [en]>])
INFO:subliminal.core:Initializing provider addic7ed
INFO:subliminal.providers.addic7ed:Logging in
DEBUG:subliminal.providers.addic7ed:Logged in
DEBUG:subliminal.providers.addic7ed:Getting show id
WARNING:subliminal.providers.addic7ed:Series not found in show ids
ERROR:subliminal.providers.addic7ed:No show id found for 'Alex Inc' ({'country': None, 'year': None})
Downloading subtitles  [####################################]  100%
DEBUG:subliminal.core:Terminating initialized providers
INFO:subliminal.core:Terminating provider addic7ed
INFO:subliminal.providers.addic7ed:Logging out
DEBUG:subliminal.providers.addic7ed:Logged out
Downloaded 0 subtitle

# subliminal --version
subliminal, version 2.0.5
```

My only hint is that perhaps the use of comma and point in the official name of the TV shows (Alex, Inc.) can put subliminal on some state but no real way for me to test.

Can someone help me for that issue?

Thanks in advance.
M.

**Comment by MoonArrow** [comment]
Just as a follow up, renaming the file to Alex,.Inc.S01E04.HDTV.x264-SVA.mkv made the trick and it was recognized. I am not sure if subliminal should or not ignore the , in search but here is the reason.

M.

**Comment by ratoaq2** [comment]
Fixed by f7e8bee3d34429c1289a8d174a98adb8a1604d0b

```
subliminal --debug download --provider addic7ed --force -l en --min-score 35 --encoding utf-8 -p addic7ed Alex.Inc.S01E04.HDTV.x264-SVA.mkv
DEBUG:subliminal.cli:Collecting path Alex.Inc.S01E04.HDTV.x264-SVA.mkv
INFO:subliminal.core:Refining video with hash
WARNING:subliminal.refiners.hash:Size is lower than 10MB: hashes not computed
INFO:subliminal.core:Refining video with metadata
INFO:subliminal.core:Refining video with tvdb
INFO:subliminal.refiners.tvdb:Searching series 'Alex Inc'
DEBUG:subliminal.refiners.tvdb:Found 1 results
DEBUG:subliminal.refiners.tvdb:Found exact match on series u'Alex, Inc.'
DEBUG:subliminal.refiners.tvdb:Found series {u'seriesId': u'204745', u'rating': u'TV-PG', u'overview': u'A guy in his mid-30s with a wife and two kids makes the crazy decision to quit his good job and dive into the brave new world of starting a business.', u'firstAired': u'2018-03-28', u'zap2itId': u'SH02696747', u'aliases': [], u'networkId': u'', u'network': u'ABC (US)', u'id': 328635, u'seriesName': u'Alex, Inc.', u'status': u'Ended', u'added': u'2017-05-16 10:29:38', u'imdbId': u'tt6466948', u'lastUpdated': 1541949574, u'addedBy': 396427, u'genre': [u'Comedy'], u'banner': u'graphical/328635-g2.jpg', u'slug': u'alex-inc', u'airsTime': u'8:30 PM', u'siteRatingCount': 2, u'airsDayOfWeek': u'Wednesday', u'runtime': u'25', u'siteRating': 5.5}
INFO:subliminal.refiners.tvdb:Getting series episode 1x4
1 video collected / 0 video ignored / 0 error
DEBUG:subliminal.refiners.tvdb:Found episode {u'seriesId': 328635, u'airsAfterSeason': None, u'dvdEpisodeNumber': None, u'productionCode': u'', u'overview': u'Alex thinks he fired the baby sitter, but the situation gets complicated when she shows up for work.', u'airedSeasonID': 745822, u'thumbAdded': u'2018-04-19 11:21:06', u'episodeName': u'The Nanny', u'airedSeason': 1, u'airsBeforeSeason': None, u'showUrl': u'', u'id': 6627151, u'dvdDiscid': u'', u'filename': u'episodes/328635/6627151.jpg', u'dvdSeason': None, u'writers': [], u'airedEpisodeNumber': 4, u'absoluteNumber': None, u'thumbHeight': u'225', u'imdbId': u'', u'lastUpdated': 1524161520, u'director': u'', u'lastUpdatedBy': 492861, u'dvdChapter': None, u'airsBeforeEpisode': None, u'language': {u'overview': u'en', u'episodeName': u'en'}, u'thumbAuthor': 450921, u'siteRatingCount': 0, u'directors': [], u'firstAired': u'2018-04-17', u'guestStars': [], u'siteRating': 0, u'thumbWidth': u'400'}
INFO:subliminal.core:Refining video with omdb
WARNING:subliminal.refiners.omdb:No apikey. Skipping omdb refiner.
INFO:subliminal.core:Listing subtitles with provider 'addic7ed' and languages set([<Language [en]>])
INFO:subliminal.core:Initializing provider addic7ed
INFO:subliminal.core:Listing subtitles with provider 'addic7ed' and languages set([<Language [en]>])
INFO:subliminal.providers.addic7ed:Getting show ids
DEBUG:subliminal.providers.addic7ed:Found 5091 show ids
DEBUG:subliminal.providers.addic7ed:Getting show id with year
DEBUG:subliminal.providers.addic7ed:Getting show id
INFO:subliminal.providers.addic7ed:Getting the page of show id 6388, season 1
DEBUG:subliminal.providers.addic7ed:Getting show id with year
DEBUG:subliminal.providers.addic7ed:Getting show id
INFO:subliminal.providers.addic7ed:Getting the page of show id 6388, season 1
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133554/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133554/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/133554/2' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/133554/3' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/133554/4' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133554/5' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133773/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133773/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/133773/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133554/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/133773/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133554/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134023/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/133554/2' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134023/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/133554/3' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134023/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/133554/4' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134023/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133554/5' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134217/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133773/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134217/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/133773/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134217/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/133773/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134217/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/133773/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134235/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134023/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134235/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134023/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134235/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134023/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134235/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134023/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134492/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134217/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134492/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134217/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134492/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134217/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134492/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134217/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134493/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134235/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134493/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134235/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134493/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134235/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134493/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134235/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134627/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134492/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134627/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134492/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134627/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134492/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134627/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134492/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134818/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134493/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134818/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134493/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134818/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134493/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134818/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134493/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/135025/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134627/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/135025/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134627/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/135025/2' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134627/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134627/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134818/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/134818/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/7/134818/2' [it]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/134818/3' [fr]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/135025/0' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/1/135025/1' [en]>
DEBUG:subliminal.providers.addic7ed:Found subtitle <Addic7edSubtitle u'updated/8/135025/2' [fr]>
INFO:subliminal.score:Computing score of <Addic7edSubtitle u'updated/1/134217/0' [en]> for video <Episode [u'Alex, Inc.', 2018, 1x4]> with {'hearing_impaired': False}
DEBUG:subliminal.score:Using scores {'hash': 359, 'series': 180, 'audio_codec': 3, 'year': 90, 'hearing_impaired': 1, 'episode': 30, 'source': 7, 'season': 30, 'video_codec': 2, 'resolution': 2, 'release_group': 15}
DEBUG:subliminal.score:Found matches set(['episode', 'title', 'series', 'season', 'year', 'release_group'])
DEBUG:subliminal.score:Adding title match equivalent
DEBUG:subliminal.score:Matched hearing_impaired
INFO:subliminal.score:Computed score 346 with final matches set(['episode', 'title', 'series', 'season', 'hearing_impaired', 'year', 'release_group'])
INFO:subliminal.score:Computing score of <Addic7edSubtitle u'updated/1/134217/1' [en]> for video <Episode [u'Alex, Inc.', 2018, 1x4]> with {'hearing_impaired': False}
DEBUG:subliminal.score:Using scores {'hash': 359, 'series': 180, 'audio_codec': 3, 'year': 90, 'hearing_impaired': 1, 'episode': 30, 'source': 7, 'season': 30, 'video_codec': 2, 'resolution': 2, 'release_group': 15}
DEBUG:subliminal.score:Found matches set(['episode', 'title', 'series', 'season', 'year', 'release_group'])
DEBUG:subliminal.score:Adding title match equivalent
INFO:subliminal.score:Computed score 345 with final matches set(['episode', 'title', 'series', 'season', 'year', 'release_group'])
INFO:subliminal.score:Computing score of <Addic7edSubtitle u'updated/1/134217/0' [en]> for video <Episode [u'Alex, Inc.', 2018, 1x4]> with {'hearing_impaired': False}
DEBUG:subliminal.score:Using scores {'hash': 359, 'series': 180, 'audio_codec': 3, 'year': 90, 'hearing_impaired': 1, 'episode': 30, 'source': 7, 'season': 30, 'video_codec': 2, 'resolution': 2, 'release_group': 15}
DEBUG:subliminal.score:Found matches set(['episode', 'title', 'series', 'season', 'year', 'release_group'])
DEBUG:subliminal.score:Adding title match equivalent
DEBUG:subliminal.score:Matched hearing_impaired
INFO:subliminal.score:Computed score 346 with final matches set(['episode', 'title', 'series', 'season', 'hearing_impaired', 'year', 'release_group'])
INFO:subliminal.score:Computing score of <Addic7edSubtitle u'updated/1/134217/1' [en]> for video <Episode [u'Alex, Inc.', 2018, 1x4]> with {'hearing_impaired': False}
DEBUG:subliminal.score:Using scores {'hash': 359, 'series': 180, 'audio_codec': 3, 'year': 90, 'hearing_impaired': 1, 'episode': 30, 'source': 7, 'season': 30, 'video_codec': 2, 'resolution': 2, 'release_group': 15}
DEBUG:subliminal.score:Found matches set(['episode', 'title', 'series', 'season', 'year', 'release_group'])
DEBUG:subliminal.score:Adding title match equivalent
INFO:subliminal.score:Computed score 345 with final matches set(['episode', 'title', 'series', 'season', 'year', 'release_group'])
INFO:subliminal.core:Downloading subtitle <Addic7edSubtitle u'updated/1/134217/0' [en]>
INFO:subliminal.providers.addic7ed:Downloading subtitle <Addic7edSubtitle u'updated/1/134217/0' [en]>
INFO:subliminal.subtitle:Guessing encoding for language en
DEBUG:subliminal.subtitle:Trying encodings ['utf-8', 'latin-1']
INFO:subliminal.subtitle:Guessed encoding utf-8
INFO:subliminal.subtitle:Guessing encoding for language en
DEBUG:subliminal.subtitle:Trying encodings ['utf-8', 'latin-1']
INFO:subliminal.subtitle:Guessed encoding utf-8
DEBUG:subliminal.core:All languages downloaded
DEBUG:subliminal.core:Terminating initialized providers
INFO:subliminal.core:Terminating provider addic7ed
INFO:subliminal.core:Saving <Addic7edSubtitle u'updated/1/134217/0' [en]> to 'Alex.Inc.S01E04.HDTV.x264-SVA.en.srt'
INFO:subliminal.subtitle:Guessing encoding for language en
DEBUG:subliminal.subtitle:Trying encodings ['utf-8', 'latin-1']
INFO:subliminal.subtitle:Guessed encoding utf-8
Downloaded 1 subtitle
```
